docs/tutorials/08_quantum_kernel_trainer.ipynb

```diff
@@ -242,11 +238,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Use the qasm simulator backend\n",
-    "backend = AerSimulator(method=\"statevector\")\n",
-    "\n",
     "# Instantiate quantum kernel\n",
-    "quant_kernel = QuantumKernel(fm, training_parameters=training_params, quantum_instance=backend)\n",
+    "quant_kernel = TrainableFidelityQuantumKernel(feature_map=fm, training_parameters=training_params)\n",
```

Comment:
The list below in `Train the Quantum Kernel` still has the QuantumKernel type

![image](https://user-images.githubusercontent.com/40241007/195886950-64a881dc-a1d3-4642-9ba3-ab07455ceff1.png)

Reply:
Replaced with "A `TrainableKernel` object"